src/collections/_documentation/error-reporting/configuration/drain-example/python.md

```diff
@@ -8,3 +8,9 @@ client = Hub.current.client
 if client is not None:
     client.close(timeout=2.0)
 ```
+
+After shutdown the current client cannot be used any more so make sure to only
+do that right before you shut down the application.
```

Comment:
This is moved here from the global context, with one important edit: "the client" => "the **current** client".